config/initializers/simple_form_bootstrap.rb

```diff
@@ -0,0 +1,30 @@
+SimpleForm.setup do |config|
```

Comment:
you have simple_form.rb and simple_form_bootstrap.rb. Please move the logic to simple_form.rb and remove simple_form_bootstrap.rb

Reply:
fixed